wagtail/admin/api/actions/unpublish.py

```diff
@@ -0,0 +1,39 @@
+from django.core.exceptions import ValidationError as DjangoValidationError
+from rest_framework import fields, status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+
+from wagtail.core.actions.unpublish_page import UnpublishPageAction
+
+from .base import APIAction
+
+
+class UnpublishPageAPIActionSerializer(Serializer):
+    set_expired = fields.BooleanField(default=False, required=False)
+    log_action = fields.BooleanField(default=True, required=False)
+    include_descendants = fields.BooleanField(default=False, required=False)
```

Comment:
I think for consistency, with copy, we should rename this to ``recursive``: https://github.com/wagtail/wagtail/blob/main/wagtail/admin/api/actions/copy.py#L19